package/MDAnalysis/coordinates/base.py

```diff
@@ -385,15 +385,19 @@ def from_coordinates(cls,
 
         return ts
 
+
     def __getstate__(self):
+        self.dt
```

Comment:
Are you just calling `self.dt` to make stuff happen in the background? If so, add a comment why this apparent no-op is here.

Reply:
This comment is not an explanaition. Totally unclear why the side effect of this line of code is that we save 'dt' as a variable. My suggestion would be 

```
The dt property is lazy loaded. We need to load it once from the _reader attached to this timestep to load the dt value.
```